crates/aiken-lang/src/test_framework.rs

```diff
@@ -457,8 +531,16 @@ impl PropertyTest {
 ///
 #[derive(Debug)]
 pub enum Prng {
-    Seeded { choices: Vec<u8>, uplc: PlutusData },
-    Replayed { choices: Vec<u8>, uplc: PlutusData },
+    Seeded {
+        choices: Vec<u8>,
+        uplc: PlutusData,
+        iteration: usize,
+    },
+    Replayed {
+        choices: Vec<u8>,
+        uplc: PlutusData,
+        iteration: usize,
+    },
 }
```

Comment:
Why is changing the PRNG necessary? I don't think that it is right? The size parameter should be holistic for an entire PRNG and influence all random choices down the line.